### DISCUSSION TITLE
The plug-in references CDN resources. If the client network is poor, it will cause CDN resource loading failure. How to use it without introducing any CDN resources?

### DISCUSSION THREAD

**Discussion by Kylin-Qing** [imzbf/md-editor-v3#678] [Q&A]
The plug-in references CDN resources. If the client network is poor, it will cause CDN resource loading failure. How to use it without introducing any CDN resources?

**Comment by imzbf** [comment]
You can refer to this example: https://imzbf.github.io/md-editor-v3/en-US/demo#%F0%9F%99%8D%F0%9F%8F%BB%E2%80%8D%E2%99%82%EF%B8%8F%20Import%20All%20Library
